format_nomad_with_env.sh

```diff
@@ -68,19 +69,38 @@ fi
 # We need to specify the database and Nomad hosts for development, but
 # not for production because we just point directly at the RDS/Nomad
 # instances.
+# Conversely, in prod we need AWS credentials and a logging config but
+# not in development.
+# We do these with multi-line environment variables so that they can
+# be formatted into development job specs.
 if [ $env != "prod" ]; then
```

Comment:
Since we are using `prod` for production, does that mean the deployment command run by circleci should be:
```
... TF_VAR__stage=prod ./deploy.sh
```
instead of
```
 ... TF_VAR__stage=produnction ./deploy.sh
```

Reply:
I don't think it will matter as far as environments go. That `TF_VAR__stage` variable is what ends up in the name of AWS resources such as `data-refinery-dev-dhu-vpc` or whatever. I don't think its actually used for anything outside of that.

However `prod` might be better than `production` for that same reason, as `data-refinery-prod-circleci-vpc` is already a fairly long name.

Reply:
Yeah, I agree that `prod` is better. Let me rebuild the deployment within circleci with this name.